script/releases/v1.0.0-slashing/1-deployContracts.s.sol

```diff
@@ -0,0 +1,187 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import {EOADeployer} from "zeus-templates/templates/EOADeployer.sol";
+import "../Env.sol";
+
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+
+/**
+ * Purpose: use an EOA to deploy all of the new contracts for this upgrade. 
+ */
+contract Deploy is EOADeployer {
+    using Env for *;
+
+    function _runAsEOA() internal override {
+        vm.startBroadcast();
+
+        /// permissions/
+
+        address[] memory pausers = new address[](3);
+        pausers[0] = Env.pauserMultisig();
+        pausers[1] = Env.opsMultisig();
+        pausers[2] = Env.executorMultisig();
+
+        deployImpl({
+            name: type(PauserRegistry).name,
+            deployedTo: address(new PauserRegistry({
+                _pausers: pausers,
+                _unpauser: Env.executorMultisig()
+            }))
+        });
+
+        deployImpl({
+            name: type(PermissionController).name,
+            deployedTo: address(new PermissionController())
+        });
+
+        deployProxy({
+            name: type(PermissionController).name,
+            deployedTo: address(new TransparentUpgradeableProxy({
+                _logic: address(Env.impl.permissionController()),
+                admin_: Env.proxyAdmin(),
+                _data: ""
+            }))
+        });
+
+        /// core/
+
+        deployImpl({
+            name: type(AllocationManager).name,
+            deployedTo: address(new AllocationManager({
+                _delegation: Env.proxy.delegationManager(),
+                _pauserRegistry: Env.impl.pauserRegistry(),
+                _permissionController: Env.proxy.permissionController(),
+                _DEALLOCATION_DELAY: Env.MIN_WITHDRAWAL_DELAY(),
+                _ALLOCATION_CONFIGURATION_DELAY: Env.ALLOCATION_CONFIGURATION_DELAY()
+            }))
+        });
+
+        deployProxy({
+            name: type(AllocationManager).name,
+            deployedTo: address(new TransparentUpgradeableProxy({
+                _logic: address(Env.impl.allocationManager()),
+                admin_: Env.proxyAdmin(),
+                _data: ""
+            }))
+        });
+
+        deployImpl({
+            name: type(AVSDirectory).name,
+            deployedTo: address(new AVSDirectory({
+                _delegation: Env.proxy.delegationManager(),
+                _pauserRegistry: Env.impl.pauserRegistry()
+            }))
+        });
+
+        deployImpl({
+            name: type(DelegationManager).name,
+            deployedTo: address(new DelegationManager({
+                _avsDirectory: Env.proxy.avsDirectory(),
```

Comment:
would be good to rebase since you will have to update constructors (avsd is removed here)